backend/src/main/resources/templates/Ansible/insightsEmailBody.html

```diff
@@ -0,0 +1,606 @@
+    h2:last-child,
+    h3:last-child,
+    h4:last-child,
+    h5:last-child,
+    h6:last-child,
+    p:last-child {
+      margin-bottom: 0;
+    }
+
+    .rh-masthead__content-title h1,
+    .rh-masthead__content-title h2,
+    .rh-masthead__content-title h3,
+    .rh-masthead__content-title h4,
+    .rh-masthead__subhead h1,
+    .rh-masthead__subhead h2,
+    .rh-masthead__subhead h3,
+    .rh-masthead__subhead h4 {
+      font-weight: 400;
+      line-height: 1;
+    }
+
+    .rh-masthead__subhead h1 {
+      font-size: 18px;
+    }
+
+    .rh-body-table,
+    .rh-body-cell {
+      height: 100% !important;
+      margin: 0;
+      padding: 0;
+      width: 100% !important;
+    }
+
+    .rh-body-table {
+      background-color: #ffffff !important;
+    }
+
+    .rh-data-table,
+    .rh-content__block .rh-data-table {
+      width: 100%;
+      background-color: #fff;
+    }
+
+    .rh-data-table thead {
+      background-color: #e7f1fa;
+    }
+
+    .rh-data-table thead th {
+      font-weight: 700;
+      text-align: left;
+    }
+
+    .rh-data-table tbody th {
+      font-weight: 500;
+      text-align: left;
+      width: 200px;
+    }
+
+    .rh-data-table td {
+      font-size: 14px;
+    }
+
+    .rh-data-table.rh-m-bordered th {
+      padding: 10px;
+      text-align: left;
+      font-size: 14px;
+      font-weight: 500;
+      /* background-color: #ededed; */
+    }
+
+    .rh-data-table.rh-m-bordered td {
+      padding: 10px;
+      text-align: left;
+    }
+
+    .rh-data-table.rh-m-bordered {
+      border-collapse: collapse;
+    }
+
+    .rh-m-bordered tbody tr {
+      border-bottom: 1px solid #b8bbbe;
+    }
+
+    .rh-m-bordered tr:nth-child(even) {
+      background-color: #fafafa;
+    }
+
+    .rh-template-container {
+      padding-top: 0;
+      padding-right: 0;
+      padding-bottom: 24px;
+      padding-left: 0;
+      width: 600px;
+    }
+
+    .rh-template-container,
+    .rh-template-container > tbody {
+      background: #ededed;
+      color: #151515;
+    }
+
+    .rh-head,
+    .rh-body,
+    .rh-footer {
+      margin: 0;
+      mso-line-height-rule: exactly;
+      font-family: 'Red Hat Text', Helvetica;
+      font-size: 16px;
+      box-sizing: border-box;
+    }
+
+    .rh-body,
+    .rh-masthead,
+    .rh-content,
+    .rh-head {
+      width: 100% !important;
+    }
+
+    .rh-head {
+      padding-bottom: 24px;
+    }
+
+    .rh-masthead__subhead {
+      background-color: #282828 !important;
+      padding-top: 16px;
+      padding-right: 24px;
+      padding-bottom: 16px;
+      padding-right: 24px;
+      color: #ffffff !important;
+      font-size: 18px;
+      text-align: center;
+    }
+
+    .rh-masthead__brand {
+      padding-top: 8px;
+    	padding-bottom: 8px;
+      background-color: #151515 !important;
+      background-repeat: repeat;
+      background-image: url(https://raw.githubusercontent.com/RedHatInsights/frontend-assets/master/src/email-assets/bg_151515.jpg);
```

Comment:
I think you should be using this url

```suggestion
      background-image: url(https://console.redhat.com/apps/frontend-assets/email-assets/bg_151515.jpg);
```